xwiki-platform-core/xwiki-platform-like/xwiki-platform-like-ui/src/main/resources/XWiki/Like/LikeTranslations.xml

```diff
@@ -47,12 +47,16 @@ like.newlike.error=Error when trying to like the page.
 like.unlike.success=The page has been unliked.
 like.unlike.error=Error when trying to unlike the page.
 like.userprofile.menu=Liked Pages
-like.button.title=Number of likes on this page: {0}
-like.button.title.like=Click to like the current page. Number of likes on this page: {0}.
-like.button.title.unlike=Click to unlike the current page. Number of likes on this page: {0}.
+like.button.title=You cannot like pages.
```

Comment:
I think it's debatable, and if it's the case it should concern all 3 like.button.title.* translations.

These translations are the famous "double purpose messages" that were the tooltips of the "multifunction button" that both showed the number of likes and allowed to click to like / unlike. Thus the tooltip contained 2 informations.
The way the case of the like action not being available was handled is that one of the messages from the tooltip was simply not present when the action was not available.

Since the function of the button is now single (only "click to like"), the message would be empty for the case when the like action is not available. I decided to keep the same translation keys as it would not be bothering if the message remains the previous one also because it didn't really completely change purpose, only partially (some of the message is still in.